components/server/src/workspace/workspace-starter.ts

```diff
@@ -1966,8 +2115,8 @@ function resolveGitpodTasks(ws: Workspace, instance: WorkspaceInstance): TaskCon
     return tasks;
 }
 
-export async function isCheckPendingFirstEnabled(user: { id: string }): Promise<boolean> {
-    return getExperimentsClientForBackend().getValueAsync("workspace_start_check_pending_first", false, {
+export async function isUseOldWorkspaceStartEnabled(user: { id: string }): Promise<boolean> {
+    return getExperimentsClientForBackend().getValueAsync("workspace_start_old", false, {
```

Comment:
would it be better to opt into the new behavior instead of opting into the old one?

Reply:
Good call. Will do :+1:

Reply:
Done.